Submitted By: Dan Nicholson <dnicholson at linuxfromscratch dot org>
Date: 2006-10-11
Initial Package Version: 21.4a
Origin: Upstream
    http://cvs.savannah.gnu.org/viewcvs/emacs/src/xterm.h?root=emacs&r2=1.164&r1=1.163
Upstream Status: Applied
Description: Fixes an issue where the AltGr button is not properly recognized.
    http://linuxfromscratch.org/pipermail/blfs-support/2006-July/060608.html

diff -pNur emacs-21.4.orig/src/xterm.h emacs-21.4/src/xterm.h
--- emacs-21.4.orig/src/xterm.h	2002-02-22 10:41:44.000000000 +0000
+++ emacs-21.4/src/xterm.h	2006-10-12 00:31:30.000000000 +0000
@@ -21,8 +21,8 @@ Boston, MA 02111-1307, USA.  */
 
 #include <X11/Xlib.h>
 #include <X11/cursorfont.h>
-#include <X11/Xutil.h>
 #include <X11/keysym.h>
+#include <X11/Xutil.h>
 #include <X11/Xatom.h>
 #include <X11/Xresource.h>
 
